moped-editor/src/app/views/forms/NewProject.jsx

```diff
@@ -0,0 +1,280 @@
+import React from 'react';
+import { useForm, Controller } from "react-hook-form";
+import { TextField, Button, Grid, Icon, InputLabel, MenuItem, Select } from "@material-ui/core";
+import { Breadcrumb } from "matx";
+import { gql, useMutation, useQuery } from "@apollo/client";
+
+const NewProject = () => {
+
+  const { register, handleSubmit, setValue, control } = useForm();
+
+  const PHASES_QUERY = gql`
+  query Phases {
+    moped_phases(order_by: {phase_name: asc}) {
+      phase_name
+    }
+  }
+`;
+
+const STATUS_QUERY = gql`
+  query Status {
+    moped_status(order_by: {status_name: asc}) {
+      status_name
+    }
+  }
+`;
+
+const FISCAL_QUERY = gql`
+  query Fiscal {
+    moped_city_fiscal_years(order_by: {fiscal_year_value: asc}) {
+      fiscal_year_value
+    }
+  }
+`;
+
+const PRIORITY_QUERY = gql`
+  query Priority {
+    moped_proj_phases(order_by: {phase_priority: asc}) {
+      phase_priority
+    }
+  }
+`;
+
+  const addNewProject = gql `
+  mutation MyMutation($project_name: String!="", $project_description: String!="", $current_phase: String!="", $project_priority: Int!="", $current_status: String!="", $eCapris_id: bpchar!="", $fiscal_year: String!="", $capitally_funded: Boolean!="", $start_date: date="") {
+    insert_moped_project(objects: {project_name: $project_name, project_description: $project_description, project_priority: $project_priority, current_phase: $current_phase, current_status: $current_status, eCapris_id: $eCapris_id, fiscal_year: $fiscal_year, capitally_funded: $capitally_funded, start_date: $start_date }) {
+      affected_rows
+      returning {
+        project_name
+        project_description
+        project_priority
+        current_phase
+        current_status
+        eCapris_id
+        fiscal_year
+        capitally_funded
+        start_date
+      }
+    }
+  }    
+ `;
```

Comment:
These are really great! I am excited! 😃 

I'm not too opinionated as to where to put queries, but as per Mateo's documentation, we are going to try to follow this pattern: https://app.gitbook.com/@atd-dts/s/moped-technical-docs/dev-guides/code-organization

You could save all those queries in a file like that, and then import them.